ocaml/networkd/bin/network_server.ml

```diff
@@ -58,7 +61,13 @@ let on_shutdown signal =
 
 let on_timer () = write_config ()
 
-let clear_state () = config := Network_config.empty_config
+let clear_state () =
+  config := Network_config.empty_config ;
+  config_writable := false
+
+let sync_state () =
```

Comment:
Can you change the name to `persist_state`? Sync can be quite a nebulous action, while persist is much more specific.